src/web/services/course.service.ts

```diff
@@ -1,9 +1,20 @@
 import { Injectable } from '@angular/core';
 import { forkJoin, Observable } from 'rxjs';
 import { map } from 'rxjs/operators';
+import { ResourceEndpoints } from '../types/api-endpoints';
 import { Course, CourseArchive, Courses,  HasResponses, JoinStatus, MessageOutput } from '../types/api-output';
 import { CourseArchiveRequest, CourseCreateRequest, CourseUpdateRequest } from '../types/api-request';
 import { HttpRequestService } from './http-request.service';
+const {
+  BIN_COURSE,
+  COURSE,
+  COURSES,
+  COURSE_ARCHIVE,
+  HAS_RESPONSES,
+  JOIN,
+  JOIN_REMIND,
+  STUDENT,
+}: typeof ResourceEndpoints = ResourceEndpoints;
```

Comment:
@xpdavid Do you think it is alright to do it this way?

Reply:
While it is verbose, maybe we can use `ResourceEndpoints.COURSE` to indicate that it's an endpoint. I am fine with the current implementation also.

Reply:
Ok! I will use `ResourceEndpoints.COURSE` then. I will refactor for the rest of the student modules to use `api-endpoints.ts`, then this PR should (hopefully) be ready. Thanks!

Reply:
Also I think I will change the enum to be singular, as per the convention for enums.